vmdb/app/models/ems_microsoft.rb

```diff
@@ -25,9 +25,14 @@ def self.raw_connect(username, password, auth_url)
     WinRM::WinRMWebService.new(auth_url, :ssl, :user => username, :pass => password, :disable_sspi => true)
   end
 
-  def self.auth_url(ipaddress, port = nil)
+  # Utilize URI::Generic#hostname to add support for IPv6 literals
+  # TODO: simplify this once https://github.com/ruby/ruby/pull/765 lands in our ruby
+  def self.auth_url(hostname, port = nil)
     port ||= 5985
-    "http://#{ipaddress}:#{port}/wsman"
+    require 'uri'
+    uri = URI::HTTP.build(:port => port, :path => "/wsman")
+    uri.hostname = hostname
```

Comment:
@chessbyte Because URI::Generic.build doesn't accept hostname until this PR:  https://github.com/ruby/ruby/pull/765

It accepts :host, but it's not ipv6 safe and blows up unless you wrap ipv6 addresses in square brackets `[::1]`, which the client shouldn't *have* to know.  

It landed in the latest ruby 2.0.0 (p643) but it's not everywhere and monkey patching it would be doable but a bit annoying since it's a long initialize method.